src/tensorlake/cli/secrets.py

```diff
@@ -98,7 +97,7 @@ def set(auth: AuthContext, secrets: str):
     resp.raise_for_status()
 
     if len(upsert_secrets) == 1:
```

Comment:
is there a reason we check for len == 1?

Reply:
Just pluralization :)

Reply:
you could just do `set <1> secret(s)`